dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -662,7 +665,7 @@ def disabled?(selector)
 end
 
 def wait_for_jquery
-  wait_with_timeout.until { @browser.execute_script("return !!$;") }
+  wait_with_timeout.until { @browser.execute_script("return (typeof $ !== 'undefined');") }
```

Comment:
I doubt this works as expected in all browsers.  In particular, Chrome has a `$` function in its console API.  Navigate to `about:blank` and in Chrome and try this:

![image](https://cloud.githubusercontent.com/assets/1615761/20415663/a3192c52-acee-11e6-870b-ee2278af2f36.png)

Instead, you could look up `$.fn.jquery` which is a version string specific to the jQuery-version of `window.$` - or you could just do `typeof jQuery` instead of `typeof $`.